test/sqllogictest/ldbc_bi.slt

```diff
@@ -2575,44 +2575,44 @@ Explained Query:
+                                                Get l2 // { arity: 5 }
                                     Get l3 // { arity: 2 }
-                                Get l2 // { arity: 3 }
+                                Project (#0{person1id}, #1{person2id}, #4) // { arity: 3 }
+                                  Get l2 // { arity: 5 }
                   Constant // { arity: 2 }
                     - (1450, 0)
       cte l3 =
         Project (#1{person2id}, #2) // { arity: 2 }
           ReadIndex on=person_knows_person person_knows_person_person1id=[*** full scan ***] // { arity: 3 }
       cte l2 =
-        Project (#0{person1id}, #1{person2id}, #4) // { arity: 3 }
-          Join on=(#2 = least(#0{person1id}, #1{person2id}) AND #3 = greatest(#0{person1id}, #1{person2id})) type=differential // { arity: 5 }
-            implementation
-              %1[#1, #0]UKK » %0:person_knows_person[greatest(#0, #1), least(#0, #1)]KK
-            ArrangeBy keys=[[greatest(#0{person1id}, #1{person2id}), least(#0{person1id}, #1{person2id})]] // { arity: 2 }
-              Project (#1{person2id}, #2) // { arity: 2 }
-                ReadIndex on=person_knows_person person_knows_person_person1id=[*** full scan ***] // { arity: 3 }
-            ArrangeBy keys=[[#1, #0]] // { arity: 3 }
-              Reduce group_by=[least(#0{person1id}, #1{person2id}), greatest(#0{person1id}, #1{person2id})] aggregates=[sum(case when (#2{parentmessageid}) IS NULL then 10 else 5 end)] // { arity: 3 }
-                Project (#0{person1id}..=#2{parentmessageid}) // { arity: 3 }
-                  Join on=(#3{containerforumid} = #4{containerforumid} = #5{id}) type=delta // { arity: 6 }
-                    implementation
-                      %0:l1 » %1[#0]UKA » %2[#0]UKA
-                      %1 » %2[#0]UKA » %0:l1[#3]K
-                      %2 » %1[#0]UKA » %0:l1[#3]K
-                    ArrangeBy keys=[[#3{containerforumid}]] // { arity: 4 }
-                      Get l1 // { arity: 4 }
-                    ArrangeBy keys=[[#0{containerforumid}]] // { arity: 1 }
-                      Distinct project=[#0{containerforumid}] // { arity: 1 }
-                        Project (#3) // { arity: 1 }
-                          Filter (#3{containerforumid}) IS NOT NULL // { arity: 4 }
-                            Get l1 // { arity: 4 }
-                    ArrangeBy keys=[[#0{id}]] // { arity: 1 }
-                      Distinct project=[#0{id}] // { arity: 1 }
-                        Project (#1) // { arity: 1 }
-                          Filter (#0{creationdate} <= 2012-11-10 00:00:00 UTC) AND (#0{creationdate} >= 2012-11-06 00:00:00 UTC) AND (#1{id}) IS NOT NULL // { arity: 4 }
-                            ReadIndex on=forum forum_id=[*** full scan ***] // { arity: 4 }
+        Join on=(#2 = least(#0{person1id}, #1{person2id}) AND #3 = greatest(#0{person1id}, #1{person2id})) type=differential // { arity: 5 }
```

Comment:
It looks like EQProp chose a different representative for the equivalence, e.g. `#2` instead of `least(#0{person1id}, #1{person2id})`, which results in a projection that is less effective (it would be sufficient to use `#0, #1, #4`, but we now we also want to use `#2` and `#3`.

Reply:
Sorry about the highlighting. It's the first line of the deletion, and the one line of insertion at the end (everything in between seems to be whitespace change that the editor hides).